<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTestingWindow</class>
 <widget class="QMainWindow" name="QTestingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>2</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="stopjs_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop JS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startjs_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Start JS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
